subsys/bluetooth/host/Kconfig

```diff
@@ -103,6 +103,36 @@ config BLUETOOTH_INTERNAL_STORAGE
 	  an internal default handler is used for this.
 
 if BLUETOOTH_CONN
+config BLUETOOTH_HOST_FLOW_CONTROL
```

Comment:
I think we should unify this with the current BLUETOOTH_CONTROLLER_TO_HOST_FC, since it doesn't make sense to have 2 separate ones? Either you build combined and they're using the same, or you're building separately and then it's a global option. So I vote for moving this up to the top-level Kconfig file and both the Host and Controller use the same.